src/domain/accounts/entities/__tests__/account.builder.ts

```diff
@@ -5,11 +5,16 @@ import { DB_MAX_SAFE_INTEGER } from '@/domain/common/constants';
 import { faker } from '@faker-js/faker';
 import { getAddress } from 'viem';
 
+// Note: regular expression is simplified because faker has limited support for regex.
+// https://fakerjs.dev/api/helpers#fromregexp
+export const accountNameSimpleRegex = /[a-zA-Z0-9]{12,20}/i;
+
 export function accountBuilder(): IBuilder<Account> {
   return new Builder<Account>()
     .with('id', faker.number.int({ max: DB_MAX_SAFE_INTEGER }))
     .with('group_id', faker.number.int({ max: DB_MAX_SAFE_INTEGER }))
     .with('address', getAddress(faker.finance.ethereumAddress()))
+    .with('name', faker.helpers.fromRegExp(accountNameSimpleRegex))
```

Comment:
Nit: what do you think about creating a specific builder for this and using that everywhere to ensure the `fromRegexp` is used?

Reply:
Done in [fc749ad](https://github.com/safe-global/safe-client-gateway/pull/2081/commits/fc749ad9d8a857bb443ac8907f647edd68ecbe66)